cdc/sink/simple_mysql_tester.go

```diff
@@ -0,0 +1,263 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Should we add this file to coverage exclude list.
https://github.com/pingcap/ticdc/blob/0138edc804f4bb16f2f62ad6486465b7ad81a2db/Makefile#L145-L146